gnome-initial-setup/gis-driver.c

```diff
@@ -904,6 +938,25 @@ emit_startup_sound (GApplication * app, gpointer user_data)
                      G_DBUS_CALL_FLAGS_NONE, -1, NULL, NULL, NULL);
 }
 
+void
+connect_to_gdm (GisDriverPrivate *priv)
+{
+  g_autoptr(GError) error = NULL;
+
+  priv->client = gdm_client_new ();
+
+  priv->greeter = gdm_client_get_greeter_sync (priv->client, NULL, &error);
+  if (!error)
```

Comment:
```suggestion
  if (error == NULL)
```

to match surrounding code – I'll apply this locally.